File: app/src/main/java/com/univreview/model/ReviewListModel.java
package com.univreview.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/**
 * Created by DavidHa on 2017. 2. 6..
 */
public class ReviewListModel implements Serializable {
    @SerializedName("review")
    public List<Review> reviews = new ArrayList<>();
    @SerializedName("reviewAvg")
    public ReviewAverage reviewAverage = new ReviewAverage();

    public Review getReview() {
        if (reviews.size() > 0) {
            return reviews.get(0);
        }
        return null;
    }

    @Override
    public String toString() {
        return "ReviewListModel{" +
                "reviews=" + reviews +
                '}';
    }
}

File: app/src/main/java/com/univreview/model/model_kotlin/ReviewListModel.kt
package com.univreview.model.model_kotlin

/**
 * Created by DavidHa on 2017. 9. 3..
 */
data class ReviewListModel(val difficultyRateAvg: Float = 0F,
                           val assignmentRateAvg: Float = 0F,
                           val attendanceRateAvg: Float = 0F,
                           val gradeRateAvg: Float = 0F,
                           val achievementRateAvg: Float = 0F,
                           val totalAvg: Float = 0F,
                           val reviews: List<Review>)